src/main/kotlin/com/team4099/robot2021/subsystems/Wheel.kt

```diff
@@ -0,0 +1,101 @@
+package com.team4099.robot2021.subsystems
+
+import com.ctre.phoenix.sensors.CANCoder
+import com.revrobotics.CANSparkMax
+import com.revrobotics.ControlType
+import com.revrobotics.SparkMax
+import com.team4099.lib.logging.Logger
+import com.team4099.lib.units.*
+import com.team4099.lib.units.base.feet
+import com.team4099.lib.units.base.inches
+import com.team4099.lib.units.derived.*
+import com.team4099.robot2021.config.Constants
+import kotlin.math.IEEErem
+import kotlin.math.withSign
+
+class Wheel(private val directionSpark: CANSparkMax, private val driveSpark: CANSparkMax,  private val encoder: CANCoder, private val zeroOffset: Angle, public val label: String) {
+
+  private val directionPID = directionSpark.pidController
+  private val drivePID = driveSpark.pidController
+
+  private val directionSensor = sparkMaxAngularMechanismSensor(directionSpark, 1.0)
+  private val driveSensor = sparkMaxLinearMechanismSensor(driveSpark, 1.0, 3.inches)
```

Comment:
wait pls idk how I missed this last time but neither of these have the correct ratios, there is a drive gear ratio and a steering ratio between the motors and the thing we care about